### PR TITLE
Conversion from gitmodules to control-repo layout 

### PR DESCRIPTION
**Pull request by alvagante**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,40 +11,45 @@ Official Support forum: [Google Groups](https://groups.google.com/forum/#!forum/
...
 * "OLD" modules (Version 1.x) are no more supported or recommended.
   They are supposed to work also on Puppet versions before 2.6
   You can give them a look using the 1.0 branch of this repo.
 
-* "NextGen" modules (Version 2.x) are currently the most used and recommended.
-  They are compatible only with Puppet version 2.6 and later.
-  You find them here as git submodules.
+* "NextGen" modules (Version 2.x) were made before the release of Puppet 3.
+  They are compatible with Puppet version 2.6 , 3 and, for most, 4.
+  You find them linked as git submodules.
```
Comment (tuxmea) [inline README.md]: It seems as if there are no submodules anymore. Did I miss something?

File: site/profile/manifests/puppet/v4/agent_tp.pp [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    file { '/usr/local/bin/puppet':
+      ensure => link,
+      target => '/opt/puppetlabs/bin/puppet',
+    }
+    file { '/usr/local/bin/facter':
+      ensure => link,
+      target => '/opt/puppetlabs/bin/facter',
+    }
+  }
+
```
Comment (tuxmea) [inline site/profile/manifests/puppet/v4/agent_tp.pp]: I would use an array and then the lambda:
`
   $links = ['puppet', 'facter', 'hiera']
   $links.each |$link| {
      file { "/opt/puppetlabs/puppet/bin/${link}":
         ensure => link,
         target => "/usr/local/bin/${link}",
      }
   }
`

File: README.md (hunk 1 above)
Comment (alvagante) [inline README.md]: In this PR the submodules are removed.

File: site/profile/manifests/puppet/v4/agent_tp.pp (hunk 2 above)
Comment (alvagante) [inline site/profile/manifests/puppet/v4/agent_tp.pp]: Absolutely, this was code supposed to be compatible with p3, but now it's possible.
I'd make the puppet code of this control repo Puppet 4 based.

**Pull request merged**